examples/minimal.cdl

```diff
@@ -0,0 +1,220 @@
+netcdf sp041_20191205T1757 {
+    dimensions:
+        N_MEASUREMENTS = 25 ;
+    variables:
+        double LATITUDE_GPS(N_MEASUREMENTS) ;
+            LATITUDE_GPS:long_name = "latitude of each GPS location" ;
+            LATITUDE_GPS:standard_name = "latitude" ;
+            LATITUDE_GPS:units = "degrees_north" ;
+            LATITUDE_GPS:_FillValue = -9999.9 ;
+            LATITUDE_GPS:valid_max = "90" ;
+            LATITUDE_GPS:valid_min = "-90" ;
+            LATITUDE_GPS:ancillary_variables = "LATITUDE_GPS_QC" ;
+        double LONGITUDE_GPS(N_MEASUREMENTS) ;
+            LONGITUDE_GPS:long_name = "longitude of each GPS location" ;
+            LONGITUDE_GPS:standard_name = "longitude" ;
+            LONGITUDE_GPS:units = "degrees_east" ;
+            LONGITUDE_GPS:_FillValue = -9999.9 ;
+            LONGITUDE_GPS:valid_max = "180" ;
+            LONGITUDE_GPS:valid_min = "-180" ;
+            LONGITUDE_GPS:ancillary_variables = "LONGITUDE_GPS_QC" ;
+        double TIME_GPS(N_MEASUREMENTS) ;
+            TIME_GPS:long_name = "time of each GPS location" ;
+            TIME_GPS:calendar = "gregorian" ;
+            TIME_GPS:units = "seconds since 1970-01-01T00:00:00Z" ;
+            TIME_GPS:_FillValue = -1. ;
+            TIME_GPS:valid_min = 1e9 ;
+            TIME_GPS:valid_max = 4e9 ;
+            TIME_GPS:ancillary_variables = "TIME_GPS_QC" ;
+        double LATITUDE(N_MEASUREMENTS) ;
+            LATITUDE:long_name = "latitude of each measurement and GPS location" ;
+            LATITUDE:standard_name = "latitude" ;
+            LATITUDE:units = "degrees_north" ;
+            LATITUDE:_FillValue = -9999.9 ;
+            LATITUDE:valid_min = "-90" ;
+            LATITUDE:valid_max = "90" ;
+            LATITUDE:ancillary_variables = "LATITUDE_GPS_QC"
```

Comment:
missing a ; at the end of this line

Reply:
changed made in the file (plus " ;" where space was missing)